### DISCUSSION TITLE
Structuring the text to align with ISO

### DISCUSSION THREAD

**Discussion by jawache** [Green-Software-Foundation/sci#167] [General]
This document gives a lot more clarity regarding how to structure standards/specs like ours.
https://www.iso.org/files/live/sites/isoorg/files/developing_standards/docs/en/model_document-rice_model.pdf

Our template mostly follows this approach but I think it has a few great comments.

1. There is a forward in which they place a disclaimer. We have one in the membership agreement that we all agreed to use for all deliverables. 

THESE MATERIALS ARE PROVIDED “AS IS.” The parties expressly disclaim any warranties (express, implied, or otherwise), including implied warranties of merchantability, non-infringement, fitness for a particular purpose, or title, related to the materials. The entire risk as to implementing or otherwise using the materials is assumed by the implementer and user. IN NO EVENT WILL THE PARTIES BE LIABLE TO ANY OTHER PARTY FOR LOST PROFITS OR ANY FORM OF INDIRECT, SPECIAL, INCIDENTAL, OR CONSEQUENTIAL DAMAGES OF ANY CHARACTER FROM ANY CAUSES OF ACTION OF ANY KIND WITH RESPECT TO THIS DELIVERABLE OR ITS GOVERNING AGREEMENT, WHETHER BASED ON BREACH OF CONTRACT, TORT (INCLUDING NEGLIGENCE), OR OTHERWISE, AND WHETHER OR NOT THE OTHER MEMBER HAS BEEN ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.

2. The forward is also where they talk about the version of the spec, in another draft spec they they have this text in the front matter:

This document is not an ISO International Standard. It is distributed for review and comment. It is subject to change without notice and may not be referred to as an International Standard.

Recipients of this draft are invited to submit, with their comments, notification of any relevant patent rights of which they are aware and to provide supporting documentation.

3. They have great advice for what to include in a scope:

The Scope is a mandatory clause which describes what the document does (“specifies”, “establishes”, “gives 
guidelines for”, “defines terms”). It shall be worded as a series of statements of fact. It shall not contain requirements 
(“shall”), recommendations (“should”) or permissions (“may”).
The Scope shall be succinct so that it can be used as an abstract for bibliographic purposes (e.g. it is used on the ISO 
website). It shall explain the applicability of the document (and, if necessary, cases where it is not applicable).

4. The headings are numbered, which makes a lot of sense since you can refer to a section with a code 3.22. I don't think we need to land on this right now, I have a feeling we'll be moving things around for a while till we are comfortable. But I think it should be something we aim to do once the structure settles down.

**Comment by atg-abhishek** [comment]
Addressed [here](https://github.com/Green-Software-Foundation/software_carbon_intensity/blob/dev/Software_Carbon_Intensity/License.md)
